.github/workflows/main.yaml

```diff
@@ -5,7 +5,7 @@ on:
     - '*'
   pull_request:
     branches:
-      - master
+      - main
```

Comment:
Plus I found the reason why the pipeline did not run in PRs. Fixed it 👍

Reply:
lol, I and my muscle memory struggle also a lot with this :-)